client/core/trade.go

```diff
@@ -473,30 +473,88 @@ func (t *trackedTrade) isRedeemable(match *matchTracker) bool {
 	return false
 }
 
+// isRefundable will be true if we have broadcasted a swap contract, the other
+// party has not executed the required follow-up action (i.e. match status shows
+// our swap is the last action on the match) AND our swap's locktime has expired.
+func (t *trackedTrade) isRefundable(match *matchTracker) bool {
+	if match.failErr != nil || match.MetaData.Proof.RefundCoin != nil {
+		return false
+	}
+
+	wallet := t.wallets.fromWallet
+	asset := t.wallets.fromAsset
+	if !wallet.unlocked() {
+		log.Errorf("cannot refund order %s, match %s, because %s wallet is not unlocked",
+			t.ID(), match.id, unbip(asset.ID))
+		return false
+	}
+
+	dbMatch, _, proof, _ := match.parts()
+	side, status := dbMatch.Side, dbMatch.Status
+
+	// hasRedeemableSwap is true if
+	// - the client is the taker on the match and the status is TakerSwapCast
+	// - the client is the maker on the match, the status is MakerSwapCast OR
+	//   status is TakerSwapCast but Taker's swap has not been redeemed. The
+	//   second case does not prevent Maker from (re-)attempting to redeem
+	//   Taker's swap; it just ensures that Maker's swap is refunded if Taker's
+	//   swap still can't be refunded **after** Maker's locktime expires.
```

Comment:
```suggestion
	//   swap still can't be redeemed **after** Maker's locktime expires.
```

Reply:
Sounds good, although I would remove "still" from that too.